spec/factories.rb

```diff
@@ -1,5 +1,20 @@
 FactoryGirl.define do
   factory :redirection do
+    sequence(:slug) { |n| "slug#{n}" }
+    sequence(:url) { |n| "http://example#{n}.com" }
+    next_id 0
+
+    after(:create) do |redirection|
+      if redirection.next_id != 0
```

Comment:
From offline conversation:

Gabe: Even if we use seeds, we still can't `create(:redirection)` without errors
Edward: What if we make the default `next` equal to `Redirection.last` since the seeds will have a couple of redirections already in place